### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by Shuja-Siddiqui**

### CODE REVIEW

File: src/pages/DeleteUser.jsx [hunk 1]
```diff
@@ -167,6 +167,7 @@ export const DeleteUser = () => {
 
     setLoading(false);
   };
+  console.log(singleUser);
```
Comment (sabeeharif) [inline src/pages/DeleteUser.jsx:170]: please remove all console log

File: src/pages/DivisionReport/index.js [hunk 2]
```diff
@@ -21,6 +21,51 @@ export const DivisionReport = () => {
...
+      "paused",
+    ];
+    postfixes.forEach((i) => {
+      prefixes.forEach((j) => {
+        // Object.values(j[0]).forEach((k) => {
+        const a = document.getElementById(`${j[0][0]}-${i}`).value;
+        const b = document.getElementById(`${j[0][1]}-${i}`).value;
+        document.getElementById(`${j[1]}-${i}`).value =
+          parseInt(a) + parseInt(b);
+        // });
```
Comment (sabeeharif) [inline src/pages/DivisionReport/index.js:61]: also remove all unnecessary comments

**Review by sabeeharif — CHANGES_REQUESTED** [review]

File: src/pages/DeleteUser.jsx (hunk 1 above)
Comment (Shuja-Siddiqui) [reply src/pages/DeleteUser.jsx:170]: removed

File: src/pages/DivisionReport/index.js (hunk 2 above)
Comment (Shuja-Siddiqui) [reply src/pages/DivisionReport/index.js:61]: done

**Review by sabeeharif — APPROVED** [review]

**Pull request merged**